examples/sentinel/README.md

```diff
@@ -93,7 +93,8 @@ jobs:
       - bash: |
           branch=$(System.PullRequest.TargetBranch)
           branch=${branch#refs/heads/}
-          git clone $(Build.Repository.Uri) --branch=${branch} --single-branch /tmp/base
+          # uncomment the --config section if using a private git repository
+          git clone ${repouri} --branch=${branch} --single-branch /tmp/base # --config http.extraheader="AUTHORIZATION: bearer $(System.AccessToken)"
```

Comment:
What is `${repouri}`? the other instance of this change had the original `$(Build.Repository.Uri)`